CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,2 @@
+- Fix: allow send multiple measures to CB in a batch (POTS /v2/op/update) instead of using multiples single request (iotagent-json#825)
```

Comment:
```suggestion
- Fix: allow send multiple measures to CB in a batch (POST /v2/op/update) and sorted by TimeInstant when possible, instead of using multiples single request (iotagent-json#825, iotagent-node-lib#1612)
```

To mimic the same at PR https://github.com/telefonicaid/iotagent-json/pull/826 (assuming both PRs are the same change)